index.js

```diff
@@ -24,21 +25,26 @@ module.exports = (app) => {
     await createConfigYML(context, {owner, repo})
     const headDate = getDate.headDate()
     const tailDate = getDate.tailDate()
-    const config = await defaultConfig
+    const config = defaultConfig
     weeklyDigest(context, {owner, repo, headDate, tailDate}, config)
   })
   createScheduler(app, {interval: interval})
   app.on('schedule.repository', async (context) => {
     console.log(`App is running as per schedule.repository`)
     app.log('Local Time: ' + moment().format())
+    const { owner, repo } = context.repo()
     const headDate = getDate.headDate()
     const tailDate = getDate.tailDate()
-    let config = await getConfig(context, 'weekly-digest.yml')
-    config = fixConfig(config)
-    console.log(`Publish Day: ${getNumDayFromLongDay(config.publishDay)}, Today: ${moment().day()}`)
-    if (moment().day() === getNumDayFromLongDay(config.publishDay)) {
-      const { owner, repo } = context.repo()
-      weeklyDigest(context, {owner, repo, headDate, tailDate}, config)
+    let hasDuplicates = checkDuplicates(context, {owner, repo, headDate})
+    if (hasDuplicates) {
+      console.log(`Weekly Digest for this week has already been published for ${owner}/${repo}`)

```

Comment:
maybe you could log the full URL to the issue that was already created?

Reply:
:+1: Doing so.

Reply:
Fixed. 😄